extensions/discovery-and-network/horizontal-pod-autoscalers.yaml

```diff
@@ -0,0 +1,295 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: horizontal-pod-autoscalers
+  namespace: kube-public
+  labels:
+    app.kubernetes.io/name: horizontal-pod-autoscalers
+    busola.io/extension: resource
+    busola.io/extension-version: '0.5'
+data:
+  dataSources: ''
+  details: |-
+    header:
+      - name: Status
+        source: $filter(status.conditions, function($v, $i, $a) {$v.status = 'True'}).type
+        widget: Labels
+    body:
+      - name: Specification
+        widget: Panel
+        children:
+          - name: Min Replicas
+            source: spec.minReplicas
+          - name: Current Replicas
+            source: status.currentReplicas
+          - name: Max Replicas
+            source: spec.maxReplicas
+          - name: Scale Target Reference
+            source: spec.scaleTargetRef
+            widget: ControlledBy
+      - name: Metrics
+        widget: Panel
+        children:
+          - name: Resource memory on Pods
+            visibility: $count($filter(status.currentMetrics, function($v, $i, $a) {$v.resource.name = "memory"})) > 0
+            source: |
+                    $join([$string($filter(status.currentMetrics, function($v, $i, $a) {
+                      $v.resource.name = "memory"})[0].resource.current.averageUtilization),'% / ',
+                          $string($filter(spec.metrics, function($v, $i, $a) {
+                      $v.resource.name = "memory"})[0].resource.target.averageUtilization),'%'])
+          - name: Resource CPU on Pods
+            visibility: $count($filter(status.currentMetrics, function($v, $i, $a) {$v.resource.name = "cpu"})) > 0
+            source: |
+                    $join([$string($filter(status.currentMetrics, function($v, $i, $a) {
+                      $v.resource.name = "cpu"})[0].resource.current.averageUtilization),'% / ',
+                          $string($single(spec.metrics, function($v, $i, $a) {
+                      $v.resource.name = "cpu"}).resource.target.averageUtilization),'%'])
+      - widget: EventList
+        filter: $matchEvents($item, $root.kind, $root.metadata.name)
+        name: Events
+        defaultType: information
+        hideInvolvedObjects: true
+  form: |-
+    - name: minReplicas
+      simple: true
+      path: spec.minReplicas
+    - name: maxReplicas
+      simple: true
+      required: true
+      path: spec.maxReplicas
+    - path: spec.behavior
+      widget: FormGroup
+      defaultExpanded: true
```

Comment:
It's unnecessary right here, maybe better option to expand scaleUp and down by default, but in my opinion this could be closed